mercantile/__init__.py

```diff
@@ -184,7 +184,9 @@ def bounds(*tile):
     xtile, ytile, zoom = tile
     a = ul(xtile, ytile, zoom)
     b = ul(xtile + 1, ytile + 1, zoom)
-    return LngLatBbox(a[0], b[1] + LL_EPSILON, b[0] - LL_EPSILON, a[1])
+    return LngLatBbox(
+        a[0] + LL_EPSILON, b[1] + LL_EPSILON, b[0] - LL_EPSILON, a[1] - LL_EPSILON
+    )
```

Comment:
@dnomadb it's only the lower right corner of the tile bounds that needs a little kick.

Reply:
I'd rather not shift the upper left corner like this. Instead, we'll do the shift inside the calculation of the tile index as I'm doing in #101.